streams/src/test/java/org/apache/kafka/streams/integration/CleanUpIntegrationTest.java

```diff
@@ -169,34 +170,34 @@ private KStreamBuilder setupTopology() {
-                                                        public KeyValue<Long, String> apply(final Long key, final String value) {
-                                                            return new KeyValue<>(key, value);
-                                                        }
-                                                    })
-                                                    .groupByKey()
-                                                    .count("global-count");
+            .map(new KeyValueMapper<Long, String, KeyValue<Long, String>>() {
+                @Override
+                public KeyValue<Long, String> apply(final Long key, final String value) {
+                    return new KeyValue<>(key, value);
+                }
+            })
+            .groupByKey()
+            .count("global-count");
         globalCounts.to(Serdes.Long(), Serdes.Long(), OUTPUT_TOPIC);
 
         final KStream<Long, Long> windowedCounts = input
-                                                       .through(INTERMEDIATE_USER_TOPIC)
-                                                       .map(new KeyValueMapper<Long, String, KeyValue<Long, String>>() {
-                                                           @Override
-                                                           public KeyValue<Long, String> apply(final Long key, final String value) {
-                                                               Utils.sleep(1000);
-                                                               return new KeyValue<>(key, value);
-                                                           }
-                                                       })
-                                                       .groupByKey()
-                                                       .count(TimeWindows.of(35).advanceBy(10), "count")
-                                                       .toStream()
-                                                       .map(new KeyValueMapper<Windowed<Long>, Long, KeyValue<Long, Long>>() {
-                                                           @Override
-                                                           public KeyValue<Long, Long> apply(final Windowed<Long> key, final Long value) {
-                                                               return new KeyValue<>(key.window().start() + key.window().end(), value);
-                                                           }
-                                                       });
+            .through(INTERMEDIATE_USER_TOPIC)
+            .map(new KeyValueMapper<Long, String, KeyValue<Long, String>>() {
+                @Override
+                public KeyValue<Long, String> apply(final Long key, final String value) {
+                    Utils.sleep(1000);
+                    return new KeyValue<>(key, value);
+                }
+            })
+            .groupByKey()
+            .count(TimeWindows.of(35).advanceBy(10), "count")
```

Comment:
How sensitive is the test to modifications of these numbers (35, 10 for windows, or 1000 for sleep)?  Are they important?  I suppose we just want to perform *some* windowed computation in order to trigger the creation of the corresponding topics etc.